app/frontend/src/store/modules/apiAccess.js

```diff
@@ -67,6 +86,68 @@ export default {
         commit('setClientStatusLoaded', false);
         console.error(error); // eslint-disable-line no-console
       }
+    },
+
+    /**
+     * @function getAcronymDetails
+     * Fetch the acronym record
+     * @param {object} context The store context
+     */
+    async getAcronymDetails({ commit, state }) {
+      try {
+        const res = await AcronymService.getAcronym(state.acronym);
+        commit('setAppName', res.data.name);
+        commit('setAppDescription', res.data.description);
+      } catch (error) {
+        console.error(error); // eslint-disable-line no-console
+      }
+    },
+
+    /**
+     * @function submitConfigForm
+     * Fetch the acronym record
+     * @param {object} context The store context
+     * @returns {boolean} whether the operation succeeded
+     */
+    async submitConfigForm({ commit, state }) {
+      const configForm = {
+        applicationAcronym: state.acronym,
+        applicationName: state.appName,
+        applicationDescription: state.appDescription,
+        clientEnvironment: state.environment,
+        commonServices: CommonServiceList
+          .filter(svc => svc.type === CommonServiceTypes.KEYCLOAK)
+          .map(svc => svc.abbreviation),
+      };
+
+      try {
+        const uniqueSeed =
+          Math.random()
+            .toString(36)
+            .substring(2) + new Date().getTime().toString(36);
+        const ephemeralRSAKey = cryptico.generateRSAKey(uniqueSeed, 1024);
+        commit('setEphemeralPasswordRSAKey', ephemeralRSAKey);
+
+        const body = {
+          configForm: configForm,
+          passwordPublicKey: cryptico.publicKeyString(ephemeralRSAKey)
+        };
+
+        const res = await KeycloakService.postConfigForm(body);
+        if (res && res.data) {
+          commit('setGeneratedClient', res.data.generatedServiceClient);
+          commit('setGeneratedPassword', res.data.generatedPassword);
+          return true;
+        } else {
+          console.error(`submitConfigForm - No response for ${JSON.stringify(configForm)}`); // eslint-disable-line no-console
+          return false;
+        }
+      }
+      catch (error) {
+        console.error(`submitConfigForm - Error occurred for ${JSON.stringify(configForm)}`); // eslint-disable-line no-console
+        console.error(error); // eslint-disable-line no-console
+        return false;
```

Comment:
Would be good to put in a //TODO note for potential global error notification handling.

Reply:
added